.circleci/config.yml

```diff
@@ -20,4 +22,16 @@ jobs:
               echo "Awesome! GCS integration tests are enabled."
             fi
       - run: make format
-      - run: make test
\ No newline at end of file
+      - run: make test
+      - run: make docker
+      - run:
+          name: Run tests on generated Docker image
+          command: |
+            docker run improbable/thanos --help
```

Comment:
I think CircleCI does not like this  ): 
```
* Error parsing config file: yaml: line 30: did not find expected key
* Cannot find a job named `build` to run in the `jobs:` section of your configuration file.
```

Reply:
hmm strange. This passes in my build pipeline. I'll have a proper look when in the office.